modules/data.atmosphere/R/download.PalEON_ENS.R

```diff
@@ -0,0 +1,60 @@
+##' Download PalEON files
+##'
+##' @name download.PalEON
+##' @title download.PalEON
+##' @export
+##' @param outfolder
+##' @param start_year
+##' @param end_year
+##' 
+##' @author Betsy Cowdery, Mike Dietze
+download.PalEON_ENS <- function(sitename, outfolder, start_date, end_date, overwrite = FALSE, ...) {
+  
+  ## parse dates
+  start_date <- as.POSIXlt(start_date, tz = "UTC")
+  end_date   <- as.POSIXlt(end_date, tz = "UTC")
+  start_year <- lubridate::year(start_date)
+  end_year   <- lubridate::year(end_date)
+  ylist      <- start_year:end_year
+  
+  ## install iCommands: 
+  ##  wget ftp://ftp.renci.org/pub/irods/releases/4.1.9/ubuntu14/irods-icommands-4.1.9-ubuntu14-x86_64.deb
+  ##  sudo dpkg -i irods-icommands-4.1.9-ubuntu14-x86_64.deb
+  ##  iinit
+  ## then follow https://pods.iplantcollaborative.org/wiki/display/DS/Setting+Up+iCommands to connect
+  
+  ## Get the data from iPlant
+  ##   iget /iplant/home/crollinson/paleon/phase3_met_drivers/test_ensemble -r -T -P
+  ##
+  ###### NEED TO AUTOMATE !!!!
+  
+  
+  ## extract the data and loop over ensemble members
+  dlpath <- "/home/carya/test_ensemble/"  ## download path
+  setwd(dlpath)
+  ens_zip <- dir(dlpath,pattern="tar.bz2",)
+  results <- list()
+  for(i in seq_along(ens_zip)){
+    system2("tar",paste("-xvjf",ens_zip[i]))  ## unzip file
```

Comment:
You actually don't even need `paste` here -- arguments are space-separated automatically. Just an FYI.